resources/js/pages/QrCodes/Index.tsx

```
@@ -0,0 +1,292 @@
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from '@/components/ui/select';
+import { show as productionOrderShow } from '@/routes/production-orders';
+import {
+    index as qrCodesIndex,
+    show as qrCodesShow,
+} from '@/routes/qr-codes';
+import type { PaginationLink } from '@/types/ui';
```

ESLint

[error] `@/types/ui` type import should occur before import of `@/routes/production-orders`

(import/order)